### ISSUE TITLE
Bug: AI doesn't favor Solar Beam, Flame Wheel and Moonlight during Harsh Sunlight

### ISSUE THREAD

**Issue by Idain** [pret/pokecrystal#807]
When checking [sunny_day_moves.asm](https://github.com/pret/pokecrystal/blob/master/data/battle/ai/sunny_day_moves.asm) I noticed that the moves Solar Beam, Flame Wheel and Moonlight aren't in the list of moves preferred by the AI during harsh sunlight:

```
SunnyDayMoves:
	db FIRE_PUNCH
	db EMBER
	db FLAMETHROWER
	db FIRE_SPIN
	db FIRE_BLAST
	db SACRED_FIRE
	db MORNING_SUN
	db SYNTHESIS
	db -1 ; end
```
The fix is simple: Include these three moves into the list, respecting how they're ordered in [move_constants.asm](https://github.com/pret/pokecrystal/blob/master/constants/move_constants.asm).

I hope this is added to [bugs_and_glitches.md](https://github.com/pret/pokecrystal/blob/master/docs/bugs_and_glitches.md).

**Comment by ShadowOne333** [comment]
This is the [sunny_day_moves.asm](https://github.com/pret/pokecrystal/blob/master/data/battle/ai/sunny_day_moves.asm) file properly fixed with the changes suggested by Idain.
If needed, I can make the PR to add this into the [bugs_and_glitches.md](https://github.com/pret/pokecrystal/blob/master/docs/bugs_and_glitches.md)

```
; AI_SMART prefers these moves during harsh sunlight.

SunnyDayMoves:
	db FIRE_PUNCH
	db EMBER
	db FLAMETHROWER
	db SOLARBEAM
	db FIRE_SPIN
	db FIRE_BLAST
	db FLAME_WHEEL
	db SACRED_FIRE
	db MORNING_SUN
	db SYNTHESIS
	db MOONLIGHT
	db -1 ; end
```

**Comment by coltongit** [comment]
Fix implemented in #849
